docs/examples/ds-ig.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ig
+  labels:
+    k8s-app: example-ig
+spec:
+  selector:
+    matchLabels:
+      name: example-ig
+  template:
+    metadata:
+      labels:
+        name: example-ig
+    spec:
+      containers:
+      - name: ig
+        # Replace this with your own image
+        image: alpine:3.17
+        securityContext:
+          privileged: true
+        # Replace this with your own commands
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          wget https://github.com/inspektor-gadget/inspektor-gadget/releases/download/v0.18.1/ig-linux-amd64-v0.18.1.tar.gz
+          tar -xzf ig-linux-amd64-v0.18.1.tar.gz ig
+          mv ig /bin/ig
+          sleep inf
+        env:
+        - name: HOST_ROOT
+          value: "/host"
+        volumeMounts:
+          - mountPath: /host
+            name: host
+          - mountPath: /run
+            name: run
+          - mountPath: /sys/kernel/debug
+            name: debugfs
+          - mountPath: /sys/fs/bpf
+            name: bpffs
```

Comment:
is `/sys/fs/bpf` actually needed by `ig`? I know we need it for ig-k8s in order to [pin the mount ns map for standard tracers](https://github.com/inspektor-gadget/inspektor-gadget/blob/9d8f024001c38d5677308b8d67289c5bcaf90d43/pkg/standardgadgets/trace/standardtracerbase.go#L89), but I don't see we're pinning any object on ig.